homeassistant/helpers/entity_component.py

```diff
@@ -131,14 +132,18 @@ def _async_setup_platform(self, platform_type, platform_config,
         scan_interval = (platform_config.get(CONF_SCAN_INTERVAL) or
                          getattr(platform, 'SCAN_INTERVAL', None) or
                          self.scan_interval)
+        scan_sequential = getattr(platform, 'SCAN_SEQUENTIAL', False)
```

Comment:
Preference:

Can we have this be `MAX_PARALLEL_UPDATES = 1` or similar, and make it an `asyncio.Semaphore`?